bin/compile_search_parser.sh

```diff
@@ -6,5 +6,10 @@ npx pegjs -o src/lib/headline_filter_parser.{js,grammar.pegjs}
 # code that we're writing and we want to keep eslint warnings at 0,
 # we're telling eslint to ignore the generated file.
 PARSER_FILE=src/lib/headline_filter_parser.js
+TMP_FILE=/tmp/headline_filter_parser.js
+
 # Prepend the generated parser code with a `eslint-disable` comment.
-sed -i '1i \/* eslint-disable */' "$PARSER_FILE"
+echo '/* eslint-disable */' > "$TMP_FILE"
```

Comment:
This sed command failed to execute on my computer. I have had issues with sed incompatibilities with linux/macOS etc, so for this specific action, "pre-pend a line to the file", I got it working this way. I'm specifically on macOS Mojave  10.14.15. I open to reverting this change as well.

Reply:
Can you tell me the output of `sed --version`? Maybe you can just install a modern GNU sed on your system?

I'd generally prefer this sed one-liner. If it have to be using a temp file, it should be done with `mktemp`, which is also hard to make platform-independent. But a static temp file can randomly make problems to some users, e.g. when they have a local CI system running.

Reply:
Ahh, I see, the local CI system that may not have a tmp dir is a good argument. I'll futz around with `brew install gsed` or something - I know that macOS default unix utils are woefully out of date.

Reply:
:+1: